docs/core/extensions/options-library-authors.md

```diff
@@ -46,13 +46,13 @@ In the preceding code, the `AddMyLibraryService`:
 
 When you author a library that exposes many options to consumers, you may want to consider requiring an `IConfiguration` parameter extension method. The expected `IConfiguration` instance should be scoped to a named section of the configuration by using the <xref:Microsoft.Extensions.Configuration.IConfiguration.GetSection%2A?displayProperty=nameWithType> function.
 
-:::code language="csharp" source="snippets/configuration/options-configparam/ServiceCollectionExtensions.cs" highlight="10,12-16":::
+:::code language="csharp" source="snippets/configuration/options-configparam/ServiceCollectionExtensions.cs" highlight="10,12-14":::
```

Comment:
Preexisting issue -- the intro text leads you to expect code with GetSection in it, but GetSection doesn't appear until a later snippet. The snippet intro text doesn't seem to describe what's going on in the immediately following snippet.